var/spack/repos/builtin/packages/cp2k/package.py

```diff
@@ -133,6 +133,8 @@ class Cp2k(MakefilePackage, CudaPackage):
     depends_on('libvori@210412:', when='@8.2:+libvori', type='build')
     depends_on('spglib', when='+spglib')
 
+    patch('sirius-7.2-build-fix.patch', when='@8.2:^sirius@7.2:')
```

Comment:
Can you add a comment on top that describes what the patch does and its provenance? Also, if the patch is already stored somewhere or is available through some endpoint, it would be preferable to use that remote location instead of storing it in Spack.

Reply:
good idea, thanks.
The patch is from here: https://github.com/cp2k/cp2k/pull/1517 but I dropped the parts not required to make it more "robust" in the sense that it also works for "cp2k@master".

Reply:
```suggestion
    # see https://github.com/cp2k/cp2k/pull/1517
    patch('sirius-7.2-build-fix.patch', when='@8.2:^sirius@7.2:')
```

Reply:
Thanks!